package.json

```diff
@@ -11,7 +11,7 @@
     "history": "^4.9.0",
     "jquery": "^3.3.1",
     "markdown-it": "^8.4.2",
-    "markdown-it-music": "github:music-markdown/markdown-it-music",
+    "markdown-it-music": "^0.0.4",
```

Comment:
I updated this in the PR here: https://github.com/music-markdown/music-markdown/pull/90/files

`^` is used for minor versions, which doesn't pick up the patch. So we have to use `~` instead.

https://docs.npmjs.com/about-semantic-versioning

Reply:
Good catch. Thanks!